api/src/utils/apiWrapper.js

```diff
@@ -0,0 +1,24 @@
+const axios = require('axios');
+
+const SERVICE_URL = 'https://memberdb-encrypt.vercel.app';
```

Comment:
consider making this an environment variable

Reply:
great idea!